packages/xstate-vue/test/UseInterpret.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <button @click="service?.send('TOGGLE')" data-testid="button">
```

Comment:
is the `?.` required here? It seems to me that `service` should just always be available here 🤔

Reply:
True!